.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ExpandedNodes": [
```

Comment:
this is vs code settings should not be committed

Reply:
Thank you, added it to my gitignore

Reply:
It will still be committed now. 
move it out of the repo to keep a copy
delete it, and push that you have deleted it. 
When PR is merged just add it back then your gitignore will ignore it

Reply:
Thank you I understand and I have deleted the files